data/language/en-GB.txt

```diff
@@ -4430,7 +4430,10 @@ STR_6118    :A gentle roller coaster for people who haven't yet got the courage
 STR_6119    :A cheap and easy to build roller coaster, but with a limited height
 STR_6120    :{BABYBLUE}New vehicle now available for {STRINGID}:{NEWLINE}{STRINGID}
 STR_6121    :{SMALLFONT}{BLACK}Extends the park's land rights all the way to the edges of the map
-
+STR_6122    :Enable quick demolish mode?
+STR_6123    :{WINDOW_COLOUR_1}This mode allows demolishing rides by pressing their name in the ride list. Are you sure you wish to enable this?{NEWLINE}{NEWLINE}Once a ride is demolished, it cannot be undone.
+STR_6124    :Confirm
```

Comment:
This is actually the first time "confirm" has been added to the game, all the others are action specific confirmation texts.